clientgen/template.gotpl

```diff
@@ -37,15 +37,15 @@ type {{ .Mutation.Name | go }} {{ .Mutation.Type | ref }}
 {{- range $model := .Operation}}
 const {{ $model.Name|go }}Query = `{{ $model.Operation }}`
 
-func (c *Client) {{ $model.Name|go }} (ctx context.Context{{- range $arg := .Args }}, {{ $arg.Variable | goPrivate }} {{ $arg.Type | ref }} {{- end }}, httpRequestOptions ...client.HTTPRequestOption) (*{{ $model.ResponseStructName | go }}, error) {
+func (c *Client) {{ $model.Name | go }} (ctx context.Context{{- range $arg := .Args }}, {{ $arg.Variable | goPrivate }} {{ $arg.Type | ref }} {{- end }}, httpRequestOptions ...client.HTTPRequestOption) (*{{ $model.ResponseStructName | go }}, error) {
 	vars := map[string]interface{}{
 	{{- range $args := .VariableDefinitions}}
 		"{{ $args.Variable }}": {{ $args.Variable | goPrivate }},
 	{{- end }}
 	}
 
     var res {{ $model.ResponseStructName | go }}
-    if err := c.Client.Post(ctx, "{{ $model.Name|go }}", {{ $model.Name|go }}Query, &res, vars, httpRequestOptions...); err != nil {
+    if err := c.Client.Post(ctx, "{{ $model.Name }}", {{ $model.Name|go }}Query, &res, vars, httpRequestOptions...); err != nil {
```

Comment:
Maybe Query should be replaced by Query/Mutation/Subscription for the relevant parts of the schema to avoid name collisions but that should not be part of this pull request